CONTRIBUTING.md

```diff
@@ -1148,3 +1150,4 @@ can be found [on the wiki][wiki-CUDW].
 [wiki-CUDW]: https://github.com/coq/coq/wiki/CoqImplementorsWorkshop
 [wiki-WG]: https://github.com/coq/coq/wiki/Coq-Working-Groups
 [YouTube]: https://www.youtube.com/channel/UCbJo6gYYr0OF18x01M4THdQ
+
```

Comment:
I'm afraid our linter requires a single newline at the end of file, and you have two.

Reply:
I removed this newline